### ISSUE TITLE
Is softmax_cross_entropy_with_logits_v2 needed ?

### ISSUE THREAD

**Issue by TheToadAlly** [jakeret/tf_unet#206]
Hello, I am wondering if I can replace softmax_cross_entropy_with_logits_v2 by softmax_cross_entropy_with_logits ?
The reason being that many versions of tensorflow don't have softmax_cross_entropy_with_logits_v2.

From my understanding, softmax_cross_entropy_with_logits_v2 is needed when backpropagation will be allowed to reach the labels themselves. I guess this isn't the case of Unet? 

Thanks

**Comment by jakeret** [comment]
If you're running on an older version of tensorflow you can change it back to `softmax_cross_entropy_with_logits`. However it is deprecated and will be removed in future versions of tensorflow
